mlrun/api/api/utils.py

```diff
@@ -48,6 +48,7 @@ def project_logs_path(project) -> Path:
 
 
 def get_obj_path(schema, path, user=""):
```

Comment:
I know you only added the last section but this function seems not documented at all, can you please add a docstring or comments for why we have all those conditions, e.g `/User` and where/when it comes from etc. Thanks

Reply:
Done.